test/e2e/extra_test.go

```diff
@@ -63,5 +63,19 @@ var _ = Describe("Deploy cluster", Label("cluster-extra-ns"), func() {
 				},
 			),
 		),
+		Entry("Advanced cluster configuration with no backup and no user", Label("ns-autoscaling"),
```

Comment:
I think the number one thing to work on is the narrative of the test so it's easier to follow.

I think this `Entry` should read something like `"Can manage advanced cluster autoscaling configuration"`.

Then your tests steps should have something like `"It won't scale down if `instanceSize` conflicts with autoscaled size"`. Potentially finding one of the tickets that refers to that issue and adding a comment.

Reply:
Thanks, i changed them please check it out again
https://github.com/commercetools/mongodb-atlas-kubernetes/pull/2/commits/ae6d00a71425b8bcf287290e67625b1e832f96b9